src/vpn-api/utils/clients.ts

```diff
@@ -32,7 +32,7 @@ import * as _ from 'lodash';
 import { apiKey, captureException } from '../../utils';
 
 import { VpnClientTrustedData } from './openvpn';
-import { pooledPostAsync } from './request';
+import { pooledRequest as request } from './request';
```

Comment:
I'd prefer this to be
```suggestion
import { pooledRequest } from './request';
```
so that the usage is clear that it's pooled